paimon-flink/paimon-flink-common/pom.xml

```diff
@@ -162,6 +162,12 @@ under the License.
             <artifactId>iceberg-data</artifactId>
             <version>${iceberg.version}</version>
             <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <artifactId>parquet-avro</artifactId>
```

Comment:
Why do this here?
It seems has no relation with this PR?

Reply:
See the first commit message